backend/main.go

```diff
@@ -79,22 +83,41 @@ func main() {
 		IdleTimeout:  60 * time.Second,
 	}
 
-	go func() {
-		err := srv.ListenAndServe()
-		if err == http.ErrServerClosed {
-			err = nil
+	terminationCtx := graceful.WithTermination(context.Background())
+
+	defer graceful.Shutdown(terminationCtx, func(err error, exitCode int) {
+		shutdownCtx := context.WithoutCancel(terminationCtx)
+
+		if wait > 0 {
```

Comment:
It looks like we don't set any value this variable. So, we could remove it at all.